docs/docs/install.md

```diff
@@ -52,28 +52,28 @@ You can install ORY Kratos using [scoop](https://scoop.sh) on Windows:
 > kratos help
 ```
 
-## Download Binaries
+## Download binaries
```

Comment:
We're using the New York Times headline capitalization as a rule of thumb everywhere. Check out: https://titlecaseconverter.com

For consistency, we should probably stay with that and fix it where we don't use it. Might also be good to document this.